samples/demos/end_to_end/main.py

```diff
@@ -32,62 +34,153 @@
 MSG_SEPARATOR = "@@@"
 MSG_AWAKE = "AWAKE"
 
+PROTOCOL_ZB = "Zigbee"
+PROTOCOL_DM = "DigiMesh"
+PROTOCOL_802 = "802.15.2"
+
+DEST_NODE_ID = "GATEWAY"
+
+
+def get_destination(xb_prot):
+    if xb_prot == PROTOCOL_ZB:
+        return xbee.ADDR_COORDINATOR
+
+    if xb_prot == PROTOCOL_DM:
```

Comment:
Maybe this could just be an else clause... Whenever it is not ZB, just run the find_gateway() code and that's it

Reply:
I left it on purpose but I can remove it.